script/configure-web-proxy

```diff
@@ -37,10 +43,12 @@ if [[ $web_proxy == "nginx" ]]; then
     echo "Setting up nginx"
     sed "s/openqa.example.com/$(hostname)/" /etc/nginx/vhosts.d/openqa.conf.template > /etc/nginx/vhosts.d/openqa.conf
     sed -i -e "s/\(^[^#]*server_name  localhost;\)/#\1/" /etc/nginx/nginx.conf
+    test -n "$web_port" && sed -i "s/listen.*$/listen $web_port;/" /etc/nginx/nginx.conf
```

Comment:
`sudo sed -i "s@:80 @:8080 @g" /etc/nginx/vhosts.d/openqa.conf`

Seems like this also needs to be updated, which I noticed after trying this in a new container.

Reply:
This line seems to duplicate #L45-#L46 no? It also transforms

```
  listen 80;
  listen [::]:80;
```

into 

```
  listen $web_port;
  listen $web_port;
```

which would be an invalid configuration.

Reply:
It did not work without it. But I guess it might silently break IPv6